lib/widgets/cards/greeting_card.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import '../../theme/theme.dart';
+
+class GreetingCard extends StatelessWidget {
+  final String greetingMessage;
+  final String userName;
+  final String profilePictureUrl;
+
+  const GreetingCard({
+    super.key,
+    required this.greetingMessage,
+    required this.userName,
+    required this.profilePictureUrl,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        CircleAvatar(
+          radius: theme.borderradius.full(theme.sizing.width.s15),
+          backgroundImage: profilePictureUrl.isNotEmpty
+              ? NetworkImage(profilePictureUrl)
+              : null,
+        ),
+        SizedBox(width: theme.sizing.width.s4),
+        Flexible(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                greetingMessage,
+                style: theme.textStyle.bodyRegular.copyWith(
+                  color: theme.colors.contrastMedium,
+                ),
+              ),
+              Text(
+                userName,
```

Comment:
only first name should be visible
basically until the code finds the first space(' ')

Reply:
done